.github/workflows/inspektor-gadget.yml

```diff
@@ -587,6 +587,7 @@ jobs:
     strategy:
```

Comment:
Great, it seems like the SPO works correctly in a 2 nodes cluster using just [one replica](https://github.com/kinvolk/inspektor-gadget/blob/7fe0cefd66c74511727b0565d1223b8f3e579032/integration/command.go#L96) ... Let's hope it keeps working like this 🤞🏽